install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+# This script downloads the latest release of run from GitHub and installs it 
+# to the current directory.
+
+# Print an error message before exiting.
+trap 'echo "Error: $0:$LINENO -- installation failed"' ERR 
+
+set -e          # Exit immediately if a command fails
+set -o pipefail # Fail a pipeline if any command fails
+
+main() {
+    # Check for the existence of a file named "run" in the current directory.
+    if [ -f run ]; then
+        echo "Error: run is already installed in the current directory."
+        echo "To reinstall, remove the existing run binary and execute this script again."
```

Comment:
This error text assumes that the thing called `run` is _our_ `run`, but we don't actually know that; please clarify.

One possibility is something like, "Error: ./run exists; refusing to overwrite it."